### PR TITLE
Fix PSR-2 compatibility issues

### PR DESCRIPTION
**Pull request by bvisness**
cc @shadowhand

### CODE REVIEW

File: src/FeedbackItem.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ private function validate()
             throw new \DomainException('All feedback items must have a body');
         }
         if ($this->link && !filter_var($this->link, FILTER_VALIDATE_URL)) {
-            throw new \DomainException('\'link\' must be a valid URL'); 
+            throw new \DomainException('\'link\' must be a valid URL');
```
Comment (ameech) [inline src/FeedbackItem.php]: Why not `"'link' must be a valid URL"`? Then you don't need to escape it. Or use ticks.

File: src/Service/TwitterService.php [hunk 2]
```diff
@@ -34,20 +36,23 @@ private function get($request, $params = []) {
         return $result;
     }
 
-    public function getTweetsSince($last_id) {
+    public function getTweetsSince($last_id)
+    {
         return $this->get('https://api.twitter.com/1.1/statuses/user_timeline.json', [
```
Comment (ameech) [inline src/Service/TwitterService.php]: Wouldn't be a bad idea have the base url in a single spot, makes it a bit easier to maintain.

File: src/Service/TwitterService.php [hunk 3]
```diff
@@ -9,12 +9,14 @@ class TwitterService
...
 
-    public function __construct(TwitterAPIExchange $twitter, $screen_name) {
+    public function __construct(TwitterAPIExchange $twitter, $screen_name)
+    {
         $this->twitter = $twitter;
         $this->screen_name = $screen_name;
     }
 
-    private function get($request, $params = []) {
+    private function get($request, $params = [])
```
Comment (ameech) [inline src/Service/TwitterService.php]: Does `$params` always take an array? If so, type hint it.

File: src/FeedbackItem.php (hunk 1 above)
Comment (bvisness) [inline src/FeedbackItem.php]: Because I wrote this a long time ago. :smiley: I'll change it.

Comment (shadowhand) [inline src/FeedbackItem.php]: I'd prefer to see backticks.

**Comment by bvisness** [comment]
Ok, tweaked. Everything to your liking?

**Comment by shadowhand** [comment]
:+1:

**Pull request merged**